### PR TITLE
Collateral Section Change

### LINKED ISSUE #70: Link to docs for timelock and bug bounty
**Timelock**
https://docs.oeth.com/governance/timelock

**Bug bounty**
https://docs.oeth.com/security-and-risks/bug-bounties

### PR DESCRIPTION
**Pull request by HrikB**
Changing the Collateral Section to not include percentage since AMO strategy causes percentages to not add up.
image

### CODE REVIEW

**Comment by matt-bullock** [comment]
Two small details

- Can you add parentheses around the ticker symbols
- The Token name should be medium weight and the ticker symbol regular font weight.

image

File: sections/homepage/Collateral.tsx [hunk 1]
```diff
@@ -1,60 +1,50 @@
...
-  const backingTokens = ["eth", "weth", "steth", "reth", "frxeth", "oeth"];
-
-  const total = parseFloat(tvl).toFixed(2);
-
-  const strategiesSorted =
-    strategies &&
-    Object.keys(strategies)
-      .sort((a, b) => strategies[a].total - strategies[b].total)
-      .reverse();
+// import React, { useState } from "react";
```
Comment (smitch88) [inline sections/homepage/Collateral.tsx:1]: Do we need these commented out pieces?

File: sections/homepage/Collateral.tsx [hunk 2]
```diff
@@ -71,204 +61,65 @@ const Collateral = ({ collateral, tvl, strategies }: CollateralProps) => {
             OETH&apos;s on-chain reserves remain liquid and available for
```
Comment (smitch88) [inline sections/homepage/Collateral.tsx:61]: A quick hack around using `&apos;s` is to just use backticks, e.g, ```{`OETH's on-chain reserves`}```

File: tailwind.config.js [hunk 3]
```diff
@@ -28,7 +28,7 @@ module.exports = {
         "table-data": "#ebecf2",
         "hover-bg": "#222329",
         "tooltip-border": "#252833",
-        subheading: "#828699",
```
Comment (smitch88) [inline tailwind.config.js:31]: I feel like we keep changing this back and forth hah. The analytics designs must have a different "dimmed/helper" text

File: sections/homepage/Collateral.tsx [hunk 4]
```diff
@@ -1,60 +1,50 @@
...
+const Collateral = ({}: CollateralProps) => {
+  const width = useViewWidth();
 
-  const tokenNames = {
-    eth: "Ether",
-    weth: "Wrapped Ether",
-    steth: "Lido Staked ETH",
-    reth: "Rocket Pool ETH",
-    frxeth: "Frax ETH",
+  const tokens = {
```
[the remark this answers is not in the document]
Comment (HrikB) [reply sections/homepage/Collateral.tsx:22]: It's a little diff. I'll move to constants.

File: sections/homepage/Collateral.tsx (hunk 2 above)
Comment (HrikB) [reply sections/homepage/Collateral.tsx:61]: Yup started doing this... will make the change here as well.

**Pull request merged**